.dockerignore

```
@@ -0,0 +1,9 @@
+.venv
+.env
+.pytest_cache
+.github
+.vscode
+ansible
+.gitignore
+.gitpod.yml
+README.md
```

One extra we might want:
```suggestion
README.md
**/__pycache__
```
Annoyingly, worth noting that .dockerignore has slightly different behaviour to .gitignore, in particular in that it doesn't ignore subfolders by default - we need to explicitly reference them with the ** prefix to mean arbitrary levels of nesting should be considered for this rule. I think at a glance that this is the only entry that would matter here, but this is one of those issues that can easily lead to weird confusion when some behaviour starts caching oddly!